addons/hr_calendar/models/hr_employee.py

```diff
@@ -0,0 +1,18 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class HrEmployeeBase(models.AbstractModel):
+    _inherit = "hr.employee.base"
+
+    def _get_calendar(self, start=None, stop=None):
```

Comment:
We talked about this and `_get_calendars` no? :thinking: 

The difference here and there is that, now `_get_calendars` gives you the calendar of the first contract, while here, you want all the calendars.. maybe we can incorporate this method and this one

Reply:
Yes You're right I nned to change that :raised_hands:

Reply:
Finally, I remove this function it wasn't really useful